MultimodalRAGWithVideos/docker_image_build/build.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+services:
+  multimodalragwithvideos:
+    build:
+      args:
+        http_proxy: ${http_proxy}
+        https_proxy: ${https_proxy}
+        no_proxy: ${no_proxy}
+      context: ../
+      dockerfile: ./Dockerfile
+    image: ${REGISTRY:-opea}/multimodalragwithvideos:${TAG:-latest}
+  multimodalragwithvideos-ui:
+    build:
+      context: ../ui
+      dockerfile: ./docker/Dockerfile
+    extends: multimodalragwithvideos
+    image: ${REGISTRY:-opea}/multimodalragwithvideos-ui:${TAG:-latest}
+  bridgetower-embedder:
+    build:
+      context: GenAIComps
+      dockerfile: comps/embeddings/multimodal/bridgetower/Dockerfile
+    extends: multimodalragwithvideos
+    image: ${REGISTRY:-opea}/bridgetower-embedder:${TAG:-latest}
+  multimodal-embedding:
+    build:
+      context: GenAIComps
+      dockerfile: comps/embeddings/multimodal/multimodal_langchain/Dockerfile
+    extends: multimodalragwithvideos
+    image: ${REGISTRY:-opea}/multimodal-embedding:${TAG:-latest}
+  multimodal-retriever-redis:
+    build:
+      context: GenAIComps
+      dockerfile: comps/retrievers/multimodal/redis/langchain/Dockerfile
+    extends: multimodalragwithvideos
+    image: ${REGISTRY:-opea}/multimodal-retriever-redis:${TAG:-latest}
```

Comment:
retriever-multimodal-redis

Reply:
@chensuyue Thanks for the feedback. I have updated docker images as you suggested following image naming rules.